src/App/templates/layout/default.html.twig

```diff
@@ -46,16 +48,16 @@
                                 <li class="nav-item dropdown">
                                     <a class="nav-link dropdown-toggle" href="#" id="pageDropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Pages</a>
                                     <div class="dropdown-menu" aria-labelledby="pageDropdown">
-                                        <a class="dropdown-item" href="{{ url('page', {action: 'home'}) }}">Home</a>
-                                        <a class="dropdown-item" href="{{ url('page', {action: 'about-us'}) }}">About Us</a>
-                                        <a class="dropdown-item" href="{{ url('page', {action: 'who-we-are'}) }}">Who We Are</a>
+                                        <a class="dropdown-item" href="{{ url('app.index') }}">Home</a>
```

Comment:
Not sure if we need this one if the logo already points to the home/index page.

Reply:
Yes, but I think we can keep it for now as an example, because the other links point to `/page` handlers. @alexmerlin